tests/S3Action_test.cpp

```
@@ -141,14 +141,39 @@ SCENARIO("Calculate the bulk action on S3 triangulations" *
+    auto const current  = S3_bulk_action_alpha_one(large_count, large_count,
+                                                   large_count, 1.0L, lambda);
+    auto const proposed = S3_bulk_action_alpha_one(
+        large_count + 1, large_count, large_count + 1, 1.0L, lambda);
+    auto const delta = mpfr_values::subtract(current, proposed);
+
+    THEN("The 256-bit values and their difference remain distinguishable.")
+    {
+      CHECK_EQ(current.get_precision(), mpfr_values::precision);
+      CHECK_EQ(proposed.get_precision(), mpfr_values::precision);
+      CHECK(mpfr_zero_p(delta.fr()) == 0);
+      CHECK_EQ(mpfr_values::to_double(current),
```

Cppcheck (2.21.0)

[style] The function 'find_cell' is never used.

(unusedFunction)